dictionaries/condensed-strokes.json

```diff
@@ -1,4 +1,6 @@
 {
+"PHAOEUT/KWREU/HREU": "mightily",
```

Comment:
@paulfioravanti I think we can leave `"PHAOEUT/KWREU/HREU": "mightily",` out as using `"KWREU": "{^y}"` would produce 'mightyly'

Reply:
`PHAOEUT/KWREU/HREU` outputs "mightily" for me, which is why I left it in. As I perform each stroke, it transitions from "might" to "mighty" to "mightily". I don't have any other special dictionaries installed in Plover. It would seem we're seeing different results somehow...?

Reply:
Good point! I was thinking of the theory of the suffix strokes themselves, without testing the entry with a clean dictionary.

Most other words that change from ending in "y" to "ily" replace the `KWREU` ending with `HREU` instead of appending it (even though `HREU` is the entry for "ly" not "ily"), for example:

```
"HAUT/KWREU": "haughty",
"HAUT/HREU": "haughtily",
```

For effectively fingerspelling the last three letters, I think I'd prefer to use the suffix strokes that spell it out (`"SKWREU": "{^i}"` and `"HREU": "{^ly}",`) than rely on orthography rules to amend the "y" to an "i". In particular, because there's a working entry that doesn't depend on orthography rules, I'd like to avoid giving people the impression that `KWREU/HREU` is a legit Plover approach for "ily" when no other entries in the dictionary use it. Does that make sense? Am I over thinking it?

Reply:
> I'd like to avoid giving people the impression that `KWREU/HREU` is a legit Plover approach for "ily" when no other entries in the dictionary use it.

This makes perfect sense to me. Not overthinking at all, and if this stroke violates the principle of least surprise, then let's get rid of it :+1: